seed/challenges/03-back-end-development-certification/back-end-development-certificate.json

```diff
@@ -10,7 +10,7 @@
         [
           "http://i.imgur.com/sKYQhdG.jpg",
           "An image of our Back End Development Certificate",
-          "This challenge will give you your verified Back End Development Certificate. Before we issue your certificate, we must verify that you have completed all of Bonfires, Ziplines and Basejumps. You must also accept our Academic Honesty Pledge. Click the button below to start this process.",
+          "This challenge will give you your verified Back End Development Certificate. Before we issue your certificate, we must verify that you have completed all of the algorithm scripting challenges, front end development projects and back end developent projects. You must also accept our Academic Honesty Pledge. Click the button below to start this process.",
           ""
```

Comment:
I didn't catch this earlier. It should just say "completed all of the back end development projects"